docs/topics/native/apple-framework.md

```diff
@@ -415,78 +381,69 @@ let field = Object().field
 let clazz = Clazz()
 clazz.member(p: 42)
 
-LibKt.forIntegers(b: 1, s: 2, i: 3, l: 4)
-LibKt.forFloats(f: 2.71, d: nil)
+HelloKt.forIntegers(b: 1, s: 2, i: 3, l: 4)
+HelloKt.forFloats(f: 2.71, d: nil)
 
-let ret = LibKt.acceptFun { "\($0) Kotlin is fun" }
+let ret = HelloKt.acceptFun { "\($0) Kotlin is fun" }
 if (ret != nil) {
-  print(ret!)
+    print(ret!)
 }
 ``` 
 
-The Kotlin code is turned into very similar looking
-code in Swift. There are some small differences, though. In Kotlin any `object` has 
-only one instance. Kotlin `object Object` now has a
-constructor in Swift, and we use the `Object()` syntax to access the only instance of it.
-The instance is always the same in Swift, so that 
-`Object() === Object()` is true. 
-Methods and property names are translated as-is. Kotlin `String` is turned into Swift `String` too.
-Swift hides `NSNumber*` boxing from us too. We can pass a Swift closure to Kotlin and call a Kotlin 
-lambda function from Swift too. 
+There are some small differences between the original Kotlin code and its Swift version. In Kotlin, any `object` has 
+only one instance. Kotlin `object Object` now has a constructor in Swift, and the `Object()` syntax is used to access
+its only instance. The instance is always the same in Swift, so that `Object() === Object()` is true.
+
+Methods and property names are translated as is. For example, Kotlin's `String` is turned into Swift's `String`. Swift
```

Comment:
"For example" seems out of place here: this is not an example of how "Methods and property names are translated as is".